### ISSUE TITLE
Hentai when all labels are 0

### ISSUE THREAD

**Issue by Majkoo** [d00ML0rDz/NsfwSpy#7]
Hey, there's a minor issue like this, when i try to classify some "tough for AI neutral images":
![image](https://user-images.githubusercontent.com/73099710/166315691-e65237ee-fabb-4227-a1f4-a5609c809dd5.png)

here's the image used for this call:
![image](https://user-images.githubusercontent.com/73099710/166316207-7952eaf1-fab6-4964-b128-49ab2aad2c04.png)

here's my piece of code using your NuGet:
```
public NsfwSpyResult Classify(IFormFile file)
    {
        var nsfwSpy = new NsfwSpy();
        using (var ms = new MemoryStream())
        {
            file.CopyTo(ms);
            var fileBytes = ms.ToArray();
            var result = nsfwSpy.ClassifyImage(fileBytes);
            
            return result;
        }
        
    }
```

**Comment by d00ML0rDz** [comment]
Just tried to replicate your problem using your image and Classify method and it looks to be classifying the image correctly to me:

![image](https://user-images.githubusercontent.com/66832778/166324048-9788a749-b635-4644-9bb9-c5aea97dd5e8.png)

Are you able to use the debugger to check that the fileBytes variable is definitely getting populated correctly?

I may look at adding some kind of exception to get thrown when you see all of the prediction values return as 0 as that usually implies the file hasn't been loaded correctly.

**Comment by Majkoo** [comment]
Yes, fileBytes were getting populated correctly.
But i think i found the reason why isnt it working.

First of all, i fricked up by sending you an image that was a screenshot of the "original" image.
I started testing it for a bit, and i found out that the problem is .webp format.
Any .webp image is classified as "Hentai" with all labels set to 0.
Any .webp image converted to .png is classified just fine.
Here are the images i checked:
[images.zip](https://github.com/d00ML0rDz/NsfwSpy/files/8612316/images.zip)

Btw, keep this project up, it is great.

**Comment by d00ML0rDz** [comment]
That makes sense. I've just done some tests and it looks like ML.NET doesn't support the .webp image format, however I'll see if this is something we can overcome by converting the file before classifying it in the near future.

I also released a new version ([v3.3.1](https://www.nuget.org/packages/NsfwSpy/3.3.1)) today that now throws a 'ClassificationFailedException' when all of the result parameters are equal to 0 to tell the user that something went wrong.

Glad that you like it and thank you for following up on this.

**Comment by d00ML0rDz** [comment]
[Version 3.4.0](https://www.nuget.org/packages/NsfwSpy/3.4.0) should now be live that will automatically convert .webp files to .png before classifying them, so hopefully that should work for your images.

If you find any more issues, just let me know 😊
